demos/glacial_isostatic_adjustment/2d_cylindrical/2d_cylindrical.py

```diff
@@ -3,12 +3,13 @@
 #
 # In this tutorial, we examine an idealised 2-D loading problem in an annulus domain.
 #
-# This example focusses on differences between running simulations in a 2-D annulus and 2-D Cartesian domain.
-# These can be summarised as follows:
+# This example focusses on differences between running simulations in a 2-D annulus
+# and 2-D Cartesian domain. These can be summarised as follows:
```

Comment:
Add our lovely new way to link back to a simpler tutorial -- i.e. your base case here.

Reply:
added